resources/drush/MODULESHORTHAND.aliases.drushrc.php

```diff
@@ -0,0 +1,43 @@
+<?php
+/**
+ * @file
+ * Drush aliases.
+ */
+
+ // Shut up phpcs.
+ $aliases = array();
+
+// Client server remote information.
+$aliases['parent'] = array(
+  'remote-host' => '123.45.67.89',
+  'remote-user' => 'user',
+  'ssh-options' => '-p 2200',
+);
+
+// Production environment.
+$aliases['prod'] = array(
+  'parent' => '@parent',
```

Comment:
The 'parent' key is deprecated. "+ $alias['parent']" is the new way: https://github.com/drush-ops/drush/blob/master/examples/example.aliases.drushrc.php#L303